architecture/admission-plug-ins.adoc

```diff
@@ -0,0 +1,26 @@
+[id="admission-plug-ins"]
+= Admission plug-ins
```

Comment:
When we describing admission, I think using controller in general and plugin in specific, please refer to the related documents from the admission controllers related documents from https://v1-17.docs.kubernetes.io/docs/reference/access-authn-authz/admission-controllers/.

Reply:
@wangke19 my understanding from a conversation with @vikram-redhat is that @mcurry-rh is happy for us to use the naming convention suggested by our Engineering team. As such, I think the terminology used in the current version of the document should be suitable based on my discussion with @sttts.